Svc/BufferManager/BufferManagerComponentImpl.cpp

```diff
@@ -203,8 +203,11 @@ namespace Svc {
         }
     }
       
-    // check some assertions
-    FW_ASSERT(bufferMem == (static_cast<U8*>(memory) + memorySize));
+    // check that the initiation pointer made it to the end of allocated space
+    U8* const CURR_PTR = bufferMem;
+    U8* const END_PTR = static_cast<U8*>(memory) + memorySize;
+    FW_ASSERT(CURR_PTR == END_PTR, reinterpret_cast<U64>(CURR_PTR), reinterpret_cast<U64>(END_PTR));
```

Comment:
These U64's should really be POINTER_CAST as U64 pointers aren't supported on smaller platforms.

Reply:
Change is made...